docs/book/src/architecture/controllers/cluster.md

```diff
@@ -5,7 +5,7 @@ The Cluster controller's main responsibilities are:
 * Setting an OwnerReference on the infrastructure object referenced in `Cluster.Spec.InfrastructureRef`.
 * Cleanup of all owned objects so that nothing is dangling after deletion.
 * Keeping the Cluster's status in sync with the infrastructure Cluster's status.
-* Creating a kubeconfig secret for [target clusters](../reference/glossary#Workload cluster).
+* Creating a kubeconfig secret for [target clusters](../../reference/glossary.html#workload-cluster).
```

Comment:
```suggestion
* Creating a kubeconfig secret for [workload clusters](../../reference/glossary.html#workload-cluster).
```

I think avoiding the use of `target cluster` here would be good, since it is very specifically meant to be used in the context of pivoting cluster-api components rather than to indicate a managed cluster the way `workload cluster` is.

Reply:
👍